### DISCUSSION TITLE
Faild to resolve loader: less-loader

### DISCUSSION THREAD

**Discussion by StayAndGo** [formilyjs/vant#8] [Q&A]
在codesandbox中打开首页提供的示例（[codesandbox](https://codesandbox.io/s/lcd2l1?file=/src/App.vue&resolutionWidth=360&resolutionHeight=675)）会报错，本地运行也是如此；在项目本地安装devDependencies后仍报错 Syntax Error: Error: Cannot find module 'less' ；

**Comment by wkgg** [comment]
需要安装less及less-loader
